app/models/payment.rb

```diff
@@ -0,0 +1,23 @@
+class Payment < ApplicationRecord
+  ### CONSTANTS
```

Comment:
I don't see any callback to update the order state...We should have a state in order reflecting payment done

Reply:
Done